src/DssSpell.t.sol

```diff
@@ -943,17 +939,27 @@ contract DssSpellTest is DssSpellTestBase {
         //    the destination address,
         //    the amount to be paid
         // Initialize the array with the number of payees
-        Payee[2] memory payees = [
-            Payee(address(usds), wallets.addr("LIQUIDITY_BOOTSTRAPPING"), 8_000_000 ether), // Note: ether is only a keyword helper
-            Payee(address(usds), wallets.addr("ECOSYSTEM_TEAM"), 3_000_000 ether) // Note: ether is only a keyword helper
+        Payee[12] memory payees = [
+            Payee(address(usds), wallets.addr("BONAPUBLICA"), 4_000 ether), // Note: ether is only a keyword helper
+            Payee(address(usds), wallets.addr("WBC"), 4_000 ether), // Note: ether is only a keyword helper
+            Payee(address(usds), wallets.addr("TANGO"), 3_400 ether), // Note: ether is only a keyword helper
+            Payee(address(usds), wallets.addr("SKY_STAKING"), 2_854 ether), // Note: ether is only a keyword helper
+            Payee(address(usds), wallets.addr("AEGIS_D"), 645 ether), // Note: ether is only a keyword helper
+            Payee(address(usds), wallets.addr("CLOAKY_KOHLA_2"), 11_140 ether), // Note: ether is only a keyword helper
+            Payee(address(usds), wallets.addr("CLOAKY_2"), 20_417 ether), // Note: ether is only a keyword helper
+            Payee(address(usds), wallets.addr("BLUE"), 54_167 ether), // Note: ether is only a keyword helper
+            Payee(address(sky), wallets.addr("CLOAKY_2"), 288_000 ether), // Note: ether is only a keyword helper
+            Payee(address(sky), wallets.addr("BLUE"), 330_000 ether), // Note: ether is only a keyword helper
+            Payee(address(usds), 0x1369f7b2b38c76B6478c0f0E66D94923421891Ba, 30_654 ether), // Note: ether is only a keyword helper
+            Payee(address(usds), 0x3300f198988e4C9C63F75dF86De36421f06af8c4, 5_927_944 ether) // Note: ether is only a keyword helper
```

Comment:
These addresses should be in the test harness, even if they are not in the chainlog.